mono/mini/Makefile.am.in

```diff
@@ -702,49 +696,36 @@ MemoryIntrinsics.dll: MemoryIntrinsics.il
 
 GENMDESC_OPTS=
 
-# we don't always use the perl impl because it's an additional
-# build dependency for the poor windows users
-# $(arch_define) is the preprocessor symbol that enables all the opcodes
-# for the specific platform in mini-ops.h
-
-if CROSS_COMPILING
-GENMDESC_PRG=perl $(srcdir)/genmdesc.pl $(arch_define) $(srcdir) $(GENMDESC_OPTS)
-else
-if WASM
-GENMDESC_PRG=perl $(srcdir)/genmdesc.pl $(arch_define) $(srcdir) $(GENMDESC_OPTS)
-else
-GENMDESC_PRG=./genmdesc $(GENMDESC_OPTS)
-endif
-endif !CROSS_COMPILING
+GENMDESC_PRG=python $(srcdir)/genmdesc.py $(target_define) $(srcdir)
```

Comment:
The command `python` can mean a recent python3 or a recent python2 depending on the linux distro / way that python is compiled. Could you please specify which interpreter you're trying to use?

Reply:
It shouldn't matter.

Reply:
that was already part of the discussion of the previous PR, alas it looks like some context got lost 😕  the script runs fine on both, python2 and 3.